transfunctions/transformer.py

```diff
@@ -150,8 +184,14 @@ def visit_FunctionDef(self, node: FunctionDef) -> Optional[Union[AST, List[AST]]
         RewriteContexts().visit(tree)
         DeleteDecorator().visit(tree)
 
-        if not tree.body[0].body:
-            tree.body[0].body.append(
+        if transfunction_decorator is None:
+            raise AliasedDecoratorSyntaxError(
+                "The transfunction decorator must have been renamed."
```

Comment:
We use the FunctionTransformer's objects in different decorators. All messages above are adaptive for it, they use smth like `f"@{decorator_name}"`.